gradle.properties

```diff
@@ -1,10 +1,10 @@
 # Gradle Project settings
 projectName = hmdb
-version=2.2.0-SNAPSHOT
+version=2.3.0
```

Comment:
According to release [procedure](https://enonic.atlassian.net/wiki/spaces/~419371546/pages/661389314/How+to+release+lib+site+or+app) version should be changed directly in master branch.

Reply:
Isn't this solved when merging?

Reply:
Same for me how it is done, can you fix it?

Reply:
Sure, I will do it